src/rdbms/mongoose_rdbms_odbc.erl

```diff
@@ -184,11 +184,16 @@ unsplit_query([QueryHead], _ParamMappers, [], QueryAcc, ParamsAcc) ->
 unsplit_query([QueryHead | QueryRest], ParamMappers, [Param | Params], QueryAcc, ParamsAcc) ->
     {{value, Mapper}, ParamMappersTail} = queue:out(ParamMappers),
     NextParamMappers = queue:in(Mapper, ParamMappersTail),
-    {InlineQuery, ODBCParam} = Mapper(Param),
+    {InlineQuery, ODBCParam} = maybe_null(Param, Mapper),
     NewQueryAcc = [InlineQuery, QueryHead | QueryAcc],
     NewParamsAcc = ODBCParam ++ ParamsAcc,
     unsplit_query(QueryRest, NextParamMappers, Params, NewQueryAcc, NewParamsAcc).
 
+maybe_null(null, _) ->
+    {"null", []};
```

Comment:
`"null"` or `<<"null">>`??

Reply:
It doesn't matter that much. It's put into iolist anyway which combines binaries and strings.

Reply:
I mean both works, and the query is prepared only once on back-end initialisation.